compiler/rustc_driver/src/lib.rs

```diff
@@ -1293,9 +1293,9 @@ pub fn init_env_logger(env: &str) {
         .with_indent_lines(true)
         .with_ansi(color_logs)
         .with_targets(true)
-        .with_wraparound(10)
-        .with_verbose_exit(true)
-        .with_verbose_entry(true)
+        //.with_wraparound(10)
```

Comment:
did you want to comment these lines out or was it a temporary change ?

Reply:
Was temporary (though I really should just make a PR for this; I find any time I do debugging, I remove these).